sphinx/ext/coverage.py

```diff
@@ -239,16 +250,46 @@ def write_py_coverage(self) -> None:
                     if undoc['funcs']:
                         op.write('Functions:\n')
                         op.writelines(' * %s\n' % x for x in undoc['funcs'])
+                        if self.config.coverage_print_missing_py_items:
```

Comment:
Is there any reason to separate configrations for c_items and py_items? If none, it is enough to add one option for such purpose.

Reply:
BTW, how about using a keyword "show" for this configuration instead of "print"?

Reply:
There is no special reason for the split. I will change it.

"show" sounds better.

Reply:
`coverage_show_missing_items` is the new option

Reply:
Sounds good!